scripts/src/java/org/oppia/android/scripts/coverage/RunCoverage.kt

```diff
@@ -102,30 +106,30 @@ class RunCoverage(
    * @return a list of lists containing coverage data for each requested test target, if
    *     the file is exempted from having a test file, an empty list is returned
    */
-  fun execute() {
+  fun execute() = runBlocking {
```

Comment:
Let's wrap just the await() bits in ``runBlocking`` instead of the whole method. It's generally a best practice in concurrency programming to minimize the time spent in a potential critical section (which is essentially all coroutines in the Kotlin world, though this best practice is _much_ more nuanced in the coroutine world for what it's worth).

Reply:
Thanks for the insights, Ben. They will be really helpful, and I'll keep them in mind when working with coroutines. 

In regard to this PR, Since the flow has now switched to sequential, `runBlocking` shouldn't be needed anymore, so I’ve removed all the asynchronous flow in the recent commits.